frontend/app/package.json

```diff
@@ -69,6 +69,7 @@
     "prism-react-renderer": "^2.3.0",
     "qs": "^6.11.2",
     "react": "^18.2.0",
+    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
There should be a corresponding change in `pnpm-lock.yaml`, no?

Reply:
sure I will update the lock file aswell